src/components/AutoEmailLink.tsx

```diff
@@ -21,14 +16,15 @@ const defaultProps = {
  *     - Else just render it inside `Text` component
  */
 
-function AutoEmailLink(props) {
+function AutoEmailLink(props: AutoEmailLinkProps) {
```

Comment:
Let's destructure props
```suggestion
function AutoEmailLink({text, style}: AutoEmailLinkProps) {
```